.travis.yml

```diff
@@ -4,5 +4,6 @@ jdk:
   - openjdk11
   - oraclejdk8
 install: /bin/true
-script: mvn install --quiet -Dgpg.skip=true -DskipTests=true
```

Comment:
What do `gpg.skip` and `skipTests` do and are we sure we want to remove those flags?

Reply:
We probably want to keep --quiet since it removes noise in the log. skipTests makes it so your CI never fails due to a test issue (it skips tests) and gpg.skip we don't care about.